tests/linearizability/traffic.go

```diff
@@ -17,48 +17,91 @@ package linearizability
 import (
 	"context"
 	"fmt"
+	"math/rand"
 	"time"
 
 	"golang.org/x/time/rate"
 )
 
 var (
-	PutGetTraffic Traffic = putGetTraffic{}
+	DefaultTraffic Traffic = readWriteSingleKey{key: "key", writes: []opChance{{operation: Put, chance: 100}}}
```

Comment:
The only comment is not sure how will you extend the operation types (e.g, delete, txn). It may be difficult to extend the lineariziablity mode.